configs/emcts1/base.cfg

```diff
@@ -1,115 +1,18 @@
-@include components/logs.cfg
-@include components/rules.cfg
-
 # EMCTS1 specific settings------------------------------------------------------------------------
 # These are settings that we added on top of the regular selfplay settings.
 
 numBots = 2
 
-# victim
-maxVisits0 = 1 # victim
+botName0 = "victim"
+maxVisits0 = 1
 searchAlgorithm0 = MCTS
 
-# adversary
-maxVisits1 = 600
+botName1 = "adversary"
+maxVisits1 = 200
 searchAlgorithm1 = EMCTS1
 
-# other options
-useAuxPolicyTarget = true
-
-# Data writing------------------------------------------------------------------------------------
-
-dataBoardLen = 19
-maxDataQueueSize = 2000
-maxRowsPerTrainFile = 25000
-maxRowsPerValFile = 5000
-firstFileRandMinProp = 0.15
-
-validationProp = 0.00
-
-# Fancy game selfplay settings--------------------------------------------------------------------
-earlyForkGameProb = 0
-earlyForkGameExpectedMoveProp = 0.025
-forkGameProb = 0
-forkGameMinChoices = 3
-earlyForkGameMaxChoices = 12
-forkGameMaxChoices = 36
-sekiForkHackProb = 0
-
-cheapSearchProb = 0
-cheapSearchVisits = 100
-cheapSearchTargetWeight = 0.0
-
-reduceVisits = false
-reduceVisitsThreshold = 0.9
-reduceVisitsThresholdLookback = 3
-reducedVisitsMin = 100
-reducedVisitsWeight = 0.1
-
-handicapAsymmetricPlayoutProb = 0
-normalAsymmetricPlayoutProb = 0
-maxAsymmetricRatio = 8.0
-
-policySurpriseDataWeight = 0.5
-valueSurpriseDataWeight = 0.1
-
-estimateLeadProb = 0.45
-switchNetsMidGame = false
-
-# Draws-------------------------------------------------------------------------------------------
-
-drawRandRadius = 0.5
-noResultStdev = 0.166666666
-
-# Search limits-----------------------------------------------------------------------------------
-
-numSearchThreads = 1
-
-# Root move selection and biases------------------------------------------------------------------
-
-chosenMoveTemperatureEarly = 0.75
-chosenMoveTemperatureHalflife = 19
-chosenMoveTemperature = 0.15
-chosenMoveSubtract = 0
-chosenMovePrune = 1
-
-rootNoiseEnabled = true
-rootDirichletNoiseTotalConcentration = 10.83
-rootDirichletNoiseWeight = 0.25
-
-rootDesiredPerChildVisitsCoeff = 2
-rootNumSymmetriesToSample = 4
-
-useLcbForSelection = true
-lcbStdevs = 5.0
-minVisitPropForLCB = 0.15
-
-# Internal params---------------------------------------------------------------------------------
-
-winLossUtilityFactor = 1.0
-staticScoreUtilityFactor = 0.00
-dynamicScoreUtilityFactor = 0.40
-dynamicScoreCenterZeroWeight = 0.25
-dynamicScoreCenterScale = 0.50
-noResultUtilityForWhite = 0.0
-drawEquivalentWinsForWhite = 0.5
-
-rootEndingBonusPoints = 0.5
-rootPruneUselessMoves = true
-
-rootPolicyTemperatureEarly = 1.25
-rootPolicyTemperature = 1.1
-
-cpuctExploration = 1.1
-cpuctExplorationLog = 0.0
-fpuReductionMax = 0.2
-rootFpuReductionMax = 0.0
-valueWeightExponent = 0.5
-subtreeValueBiasFactor = 0.35
-subtreeValueBiasWeightExponent = 0.8
+# Comment this out to enable recursive victim modeling (A-MCTS-R)
```

Comment:
:+1: to making these off by deafult

Reply:
Ok, doing that will require a `KataGo-custom` PR. I'll add it to the kanban but it's out of scope for this PR

Reply:
Thanks, agree this is out of scope.